xontrib/autojump.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import builtins
+import platform
+import subprocess
+from xonsh import dirstack
+from xonsh.events import events
+
+
+__all__ = ('_autojump', '_autojump_completer')
+
+_env = builtins.__xonsh_env__
+
+
+def _get_error_path():
+    if platform.system() == "Darwin":
+        return os.path.join(_env['HOME'], "Library/autojump/errors.log")
+    elif "XDG_DATA_HOME" in _env:
+        return os.path.join(_env['XDG_DATA_HOME'], "autojump/errors.log")
+    else:
+        return os.path.join(_env['HOME'], ".local/share/autojump/errors.log")
+
+
+_env['AUTOJUMP_ERROR_PATH'] = _get_error_path()
+_env['AUTOJUMP_SOURCED'] = 1
```

Comment:
It might be worth adding ensurers, docs, and defaults to the Env object for these two.   Also, if `AUTOJUMP_SOURCED` is supposed to be used a sboolean, it should be a boolean.